apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx

```diff
@@ -47,43 +53,56 @@ const styles = {
 class TwoColumnActionBlock extends Component {
   static propTypes = {
     isRtl: PropTypes.bool.isRequired,
-    responsive: PropTypes.instanceOf(Responsive).isRequired,
+    responsive: PropTypes.instanceOf(Responsive),
     imageUrl: PropTypes.string.isRequired,
-    heading: PropTypes.string.isRequired,
-    subHeading: PropTypes.string.isRequired,
+    heading: PropTypes.string,
+    subHeading: PropTypes.string,
     description: PropTypes.string.isRequired,
-    buttonUrl: PropTypes.string.isRequired,
-    buttonText: PropTypes.string.isRequired
+    buttons: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
does this buttons prop also need to be updated for uses of `LocalClassActionBlock` and the stories?

Reply:
At this point, the only direct uses of `TwoColumnActionBlock` are in the same file and have already been updated.